### ISSUE TITLE
Timeout doesn't work with minified version

### ISSUE THREAD

**Issue by freshface** [mathiasbynens/jquery-slideshow#1]
Hi

When i set the timeout option and used the min. version of the plugin it always takes the 3e3 timeout and not the one i set via the option constructor.

Also when there is only 1 image in the slideshow it shot not play.

Regards

**Comment by mathiasbynens** [comment]
Fix typo in minified version. Closed by 8779ab70b1d6cced37f89e628c7b47ca0c4cb031.

**Comment by mathiasbynens** [comment]
D'oh, silly typo in the minified version. That’s what I get for minifying by hand ;)

> When there is only 1 image, it should not play.

That’s easy enough to do in your own code — something like:

```
var $slideshow = $('.slideshow');
if ($slideshow.children().length > 1) {
  $slideshow.slideshow({ speed: 300, timeout: 4000 });
}
```

I’m not sure if this should be added to the plugin itself.

Thanks for reporting!
